pkg/tree/builder/gentoo/simpleparser.go

```diff
@@ -158,27 +164,46 @@ func ParseRDEPEND(rdepend string) (*GentooRDEPEND, error) {
 				continue
 			}
 
+			if strings.HasPrefix(rr, "|| (") {
```

Comment:
I'm wondering if this should be resilent e.g. to spaces.. wdyt about looking after  ```||``` ```()``` etc. ?